src/YahooFinance.py

```diff
@@ -9,12 +9,12 @@ class YahooFinanceQuote:
   # Expects the ticker symbol as the only argument.
```

Comment:
It doesn't look like we have any real changes in this file. Can we just revert it back to HEAD?